pom.xml

```diff
@@ -234,6 +234,13 @@
 
 	<dependencies>
 		<dependency>
+			<groupId>org.atmosphere</groupId>
+			<artifactId>atmosphere-runtime</artifactId>
+			<version>2.4.12</version>
+			<scope>provided</scope>
```

Comment:
@stiemannkj1 Please verify that `<scope>provided</scope>` is necessary/correct. In the 4.x branch of liferay-faces-bridge-impl/bridge-impl/pom.xml we don't specify a scope for commons-fileupload. In other words, I think `<optional>true</optional>` with `<scope>compile</scope>` should be fine.